src/build_tools/package.py

```diff
@@ -16,7 +16,7 @@
 @author: Alex Domingo (Vrije Universiteit Brussel)
 """
 
-VERSION = '4.2.3'
+VERSION = '4.2.4'
```

Comment:
another PR got merged in the mean time :)
```suggestion
VERSION = '4.2.5'
```

Reply:
fixed in https://github.com/vub-hpc/build_tools/pull/47/commits/291a0883fbb62bcde69440a7b9cc20ef59f56d94